docs/developer/payments/transactions.mdx

```diff
@@ -412,6 +412,12 @@ Query variables:
 }
 ```
 
+:::info
```

Comment:
suggestion: I think we should raise severity here, to avoid confusion with "why my order is missing"? 

```suggestion
:::warning
```